spec/requests/guide_cards_spec.rb

```diff
@@ -3,17 +3,25 @@
 require 'rails_helper'
 
 RSpec.describe 'GuideCards', type: :request do
+  before do
+    GuideCardLoadingService.new.import
+  end
   describe 'GET /index' do
     it 'returns http success' do
       get '/guide_cards/index'
       expect(response).to have_http_status(:success)
+      # expect(response.body).to include('GuideCardLoadingService')
```

Comment:
@leefaisonr I would think the thing to expect here is the heading of the five records in the sample data. 

```
      expect(response.body).to include('AALAS')
      expect(response.body).to include(WHATEVER THE SECOND OBJECT HEADING IS)
      expect(response.body).to include(WHATEVER THE THIRD OBJECT HEADING IS)
```